src/api.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+export const getUsers = () => {
+  return axios.get('https://nc-news-fnav.onrender.com/api/users').then((user)=> {
+        return user.data
+    })
+    .catch()
+}
+
+export const getArticles = (sortby, orderby) => {
+  
+  const params = {
+    // topic: topic|| undefined,
+    sortBy: sortby|| undefined,
+    orderBy: orderby || undefined,
+    // author: author || undefined
+  }
+
+
+  return axios.get('https://nc-news-fnav.onrender.com/api/articles', { params }).then((articles)=>{
+  
+  return articles.data
+  })
```

Comment:
Good job for thinking ahead! The functionality for providing queries to this input shouldn't be necessary for task 4 yet, though.
There's more functionality here than you need :) In the real world, not working on the ticket you've currently been given can get confusing, as it's difficult to trace the necessary bits of code when somebody will inevitably review task 10 later on :)